ensemble/volren/volume_axes.py

```diff
@@ -20,6 +20,16 @@ class VolumeAxes(ABCVolumeSceneMember):
     # Which axes should have a scale shown?
     visible_axis_scales = Tuple(Bool, Bool, Bool)
 
+    # Axes titles.
+    axis_titles = Tuple(Str, Str, Str)
```

Comment:
Thanks.  It would be useful to have a `StrIReallyMeanIt` trait that doesn't accept Unicode, both for documentation purposes and for early catching of this sort of error.